.github/ISSUE_TEMPLATE.md

```diff
@@ -22,7 +22,7 @@
   and `npm start`.
 
   We have a simple online template for you to use as a base to explain your bug or issue:
-  https://jsfiddle.net/vvoyer/88e40wtv/4/
+  https://jsfiddle.net/bobylito/9h7sgo10/
```

Comment:
Isn't there missing a `/` ?

-> http://jsfiddle.net/bobylito/9h7sgo10/

Reply:
I don't see the difference between your link and the one listed above 🤔

Reply:
> Isn't there missing a / ?

This is a chrome extension that is failing, I have it also but the actual content is the good one.

![image](https://user-images.githubusercontent.com/123822/31941455-3085b30a-b8c2-11e7-8fdc-52549b13d7de.png)

Reply:
Nice catch @vvo 👍